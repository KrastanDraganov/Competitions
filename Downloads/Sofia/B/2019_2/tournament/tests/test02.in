10
6 1 2 4 5 6 8
7 2 3 4 5 6 7 8
0
4 0 2 5 8
6 2 3 5 7 8 9
1 2
7 2 3 4 5 7 8 9
5 0 2 3 5 8
2 2 5
7 0 1 2 3 5 7 8
