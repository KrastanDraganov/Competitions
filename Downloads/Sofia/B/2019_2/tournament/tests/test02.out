6 4 9 7 3 0 1 8 5 2 
